framework/src/media/voice/SpeechDetector.cpp

```diff
@@ -61,24 +61,23 @@ bool SpeechDetectorImpl::initKeywordDetect(uint32_t samprate, uint8_t channels)
 		return false;
 	}
 
-	int card = -1;
-	int device = -1;
+	int sd_card = -1;
+	int sd_device = -1;
 
 	audio_manager_result_t result = find_stream_in_device_with_process_type(
 		AUDIO_DEVICE_PROCESS_TYPE_SPEECH_DETECTOR,
 		AUDIO_DEVICE_SPEECH_DETECT_KD,
-		&card, &device);
+		&sd_card, &sd_device);
 
 	if (result == AUDIO_MANAGER_SUCCESS) {
-		medvdbg("KeywordDetector : card_id : %d device_id %d\n", card, device);
-		mKeywordDetector = std::make_shared<HardwareKeywordDetector>(card, device);
-
-		result = change_stream_in_device(card, device);
-		if (result != AUDIO_MANAGER_SUCCESS && result != AUDIO_MANAGER_DEVICE_ALREADY_IN_USE) {
-			meddbg("change_stream_in_device failed: %d\n", result);
-			mKeywordDetector = nullptr;
-			return false;
-		}
+		medvdbg("KeywordDetector : card_id : %d device_id %d\n", sd_card, sd_device);
```

Comment:
Please add description about this. 'why this is needed?'

Reply:
finding AUDIO_DEVICE_PROCESS_TYPE_NONE will be implemented later